### DISCUSSION TITLE
Dispersion Correction

### DISCUSSION THREAD

**Discussion by achyuttt** [dftbplus/dftbplus#1450] [Q&A] [answered]
I have a question about Dispersion correction!

Does DFTB+ implement  dispersion correction to account for weak interaction?
If yes, how is it done? 

Please let me know if anyone of us know it

**Comment by aradi** [comment, marked as answer]
It does. Just search for ``dispersion`` in the [DFTB+ reference manual](https://dftbplus.org/documentation.html).
